crates/uv-cli/src/lib.rs

```diff
@@ -5862,6 +5862,37 @@ pub struct PythonDirArgs {
     pub bin: bool,
 }
 
+#[derive(Args)]
+pub struct PythonInstallCompileBytecodeArgs {
+    /// Compile Python's standard library to bytecode after installation.
+    ///
+    /// By default, uv does not compile Python (`.py`) files to bytecode (`__pycache__/*.pyc`);
+    /// instead, compilation is performed lazily the first time a module is imported. For use-cases
+    /// in which start time is critical, such as CLI applications and Docker containers, this option
```

Comment:
I'd put something other than "critical", like "For cases where first-start time is important"